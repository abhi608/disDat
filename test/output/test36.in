Starting T1
Starting T2
Starting read only transaction T3
=== Site 1 ===
All other variables have same initial value
=== Site 2 ===
All other variables have same initial value
=== Site 3 ===
All other variables have same initial value
=== Site 4 ===
All other variables have same initial value
=== Site 5 ===
All other variables have same initial value
=== Site 6 ===
All other variables have same initial value
=== Site 7 ===
All other variables have same initial value
=== Site 8 ===
All other variables have same initial value
=== Site 9 ===
All other variables have same initial value
=== Site 10 ===
All other variables have same initial value
Site: 1 failed
Site: 3 failed
T2 got write lock on x3
Site: 4 failed
T2 aborted as site 4 failed
Site: 4 recovered
=== Site 1 ===
This site is down
=== Site 2 ===
All other variables have same initial value
=== Site 3 ===
This site is down
=== Site 4 ===
x20 is not available for reading
x3: 30 (available at site 4 for reading as it is the only copy or has been written after recovery)
x18 is not available for reading
x14 is not available for reading
x12 is not available for reading
x16 is not available for reading
x13: 130 (available at site 4 for reading as it is the only copy or has been written after recovery)
x10 is not available for reading
x4 is not available for reading
x2 is not available for reading
x8 is not available for reading
x6 is not available for reading
=== Site 5 ===
All other variables have same initial value
=== Site 6 ===
All other variables have same initial value
=== Site 7 ===
All other variables have same initial value
=== Site 8 ===
All other variables have same initial value
=== Site 9 ===
All other variables have same initial value
=== Site 10 ===
All other variables have same initial value
=== Site 1 ===
This site is down
=== Site 2 ===
All other variables have same initial value
=== Site 3 ===
This site is down
=== Site 4 ===
x20 is not available for reading
x3: 30 (available at site 4 for reading as it is the only copy or has been written after recovery)
x18 is not available for reading
x14 is not available for reading
x12 is not available for reading
x16 is not available for reading
x13: 130 (available at site 4 for reading as it is the only copy or has been written after recovery)
x10 is not available for reading
x4 is not available for reading
x2 is not available for reading
x8 is not available for reading
x6 is not available for reading
=== Site 5 ===
All other variables have same initial value
=== Site 6 ===
All other variables have same initial value
=== Site 7 ===
All other variables have same initial value
=== Site 8 ===
All other variables have same initial value
=== Site 9 ===
All other variables have same initial value
=== Site 10 ===
All other variables have same initial value
DEBUG| Clearing site locks for T2 variable: x3
Clearing locks for T2 variable: x3
Starting T4
Starting T5
Starting read only transaction T6
Site: 2 failed
Site: 4 failed
Site: 5 failed
Site: 6 failed
Site: 7 failed
Site: 8 failed
Site: 9 failed
T5 committed
=== Site 1 ===
This site is down
=== Site 2 ===
This site is down
=== Site 3 ===
This site is down
=== Site 4 ===
This site is down
=== Site 5 ===
This site is down
=== Site 6 ===
This site is down
=== Site 7 ===
This site is down
=== Site 8 ===
This site is down
=== Site 9 ===
This site is down
=== Site 10 ===
All other variables have same initial value
Site: 4 recovered
Site: 2 failed
=== Site 1 ===
This site is down
=== Site 2 ===
This site is down
=== Site 3 ===
This site is down
=== Site 4 ===
x20 is not available for reading
x3: 30 (available at site 4 for reading as it is the only copy or has been written after recovery)
x18 is not available for reading
x14 is not available for reading
x12 is not available for reading
x16 is not available for reading
x13: 130 (available at site 4 for reading as it is the only copy or has been written after recovery)
x10 is not available for reading
x4 is not available for reading
x2 is not available for reading
x8 is not available for reading
x6 is not available for reading
=== Site 5 ===
This site is down
=== Site 6 ===
This site is down
=== Site 7 ===
This site is down
=== Site 8 ===
This site is down
=== Site 9 ===
This site is down
=== Site 10 ===
All other variables have same initial value
Starting T7
T6 committed
=== Site 1 ===
This site is down
=== Site 2 ===
This site is down
=== Site 3 ===
This site is down
=== Site 4 ===
x20 is not available for reading
x3: 30 (available at site 4 for reading as it is the only copy or has been written after recovery)
x18 is not available for reading
x14 is not available for reading
x12 is not available for reading
x16 is not available for reading
x13: 130 (available at site 4 for reading as it is the only copy or has been written after recovery)
x10 is not available for reading
x4 is not available for reading
x2 is not available for reading
x8 is not available for reading
x6 is not available for reading
=== Site 5 ===
This site is down
=== Site 6 ===
This site is down
=== Site 7 ===
This site is down
=== Site 8 ===
This site is down
=== Site 9 ===
This site is down
=== Site 10 ===
All other variables have same initial value
