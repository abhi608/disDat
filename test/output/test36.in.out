Starting T1
Starting T2
Starting read only transaction T3
=== Site 1 ===
x20: 200 at site 1
x16: 160 at site 1
x14: 140 at site 1
x10: 100 at site 1
x8: 80 at site 1
x6: 60 at site 1
x4: 40 at site 1
x18: 180 at site 1
x12: 120 at site 1
x2: 20 at site 1
=== Site 2 ===
x20: 200 at site 2
x16: 160 at site 2
x14: 140 at site 2
x8: 80 at site 2
x6: 60 at site 2
x11: 110 at site 2
x4: 40 at site 2
x18: 180 at site 2
x12: 120 at site 2
x2: 20 at site 2
x10: 100 at site 2
x1: 10 at site 2
=== Site 3 ===
x20: 200 at site 3
x16: 160 at site 3
x14: 140 at site 3
x10: 100 at site 3
x8: 80 at site 3
x6: 60 at site 3
x4: 40 at site 3
x18: 180 at site 3
x12: 120 at site 3
x2: 20 at site 3
=== Site 4 ===
x20: 200 at site 4
x16: 160 at site 4
x14: 140 at site 4
x10: 100 at site 4
x6: 60 at site 4
x4: 40 at site 4
x13: 130 at site 4
x8: 80 at site 4
x3: 30 at site 4
x18: 180 at site 4
x12: 120 at site 4
x2: 20 at site 4
=== Site 5 ===
x20: 200 at site 5
x16: 160 at site 5
x14: 140 at site 5
x10: 100 at site 5
x8: 80 at site 5
x6: 60 at site 5
x4: 40 at site 5
x18: 180 at site 5
x12: 120 at site 5
x2: 20 at site 5
=== Site 6 ===
x20: 200 at site 6
x14: 140 at site 6
x15: 150 at site 6
x10: 100 at site 6
x8: 80 at site 6
x6: 60 at site 6
x16: 160 at site 6
x5: 50 at site 6
x4: 40 at site 6
x18: 180 at site 6
x12: 120 at site 6
x2: 20 at site 6
=== Site 7 ===
x20: 200 at site 7
x16: 160 at site 7
x14: 140 at site 7
x10: 100 at site 7
x8: 80 at site 7
x6: 60 at site 7
x4: 40 at site 7
x18: 180 at site 7
x12: 120 at site 7
x2: 20 at site 7
=== Site 8 ===
x17: 170 at site 8
x16: 160 at site 8
x14: 140 at site 8
x10: 100 at site 8
x8: 80 at site 8
x20: 200 at site 8
x7: 70 at site 8
x6: 60 at site 8
x4: 40 at site 8
x18: 180 at site 8
x12: 120 at site 8
x2: 20 at site 8
=== Site 9 ===
x20: 200 at site 9
x16: 160 at site 9
x14: 140 at site 9
x10: 100 at site 9
x8: 80 at site 9
x6: 60 at site 9
x4: 40 at site 9
x18: 180 at site 9
x12: 120 at site 9
x2: 20 at site 9
=== Site 10 ===
x16: 160 at site 10
x19: 190 at site 10
x14: 140 at site 10
x10: 100 at site 10
x20: 200 at site 10
x9: 90 at site 10
x8: 80 at site 10
x6: 60 at site 10
x4: 40 at site 10
x18: 180 at site 10
x12: 120 at site 10
x2: 20 at site 10
Site: 1 failed
Site: 3 failed
T2 got write lock on x3
Site: 4 failed
T2 aborted as site 4 failed
DEBUG| Clearing site locks for T2 variable: x3
Clearing locks for T2 variable: x3
T1 got read lock on x2 having value 20
T1 got write lock on x2
Site: 4 recovered
=== Site 1 ===
This site is down
=== Site 2 ===
x20: 200 at site 2
x16: 160 at site 2
x14: 140 at site 2
x8: 80 at site 2
x6: 60 at site 2
x11: 110 at site 2
x4: 40 at site 2
x18: 180 at site 2
x12: 120 at site 2
x2: 20 at site 2
x10: 100 at site 2
x1: 10 at site 2
=== Site 3 ===
This site is down
=== Site 4 ===
x20 is not available for reading
x16 is not available for reading
x14 is not available for reading
x10 is not available for reading
x6 is not available for reading
x4 is not available for reading
x13: 130 (available at site 4 for reading as it is the only copy or has been written after recovery)
x8 is not available for reading
x3: 30 (available at site 4 for reading as it is the only copy or has been written after recovery)
x18 is not available for reading
x12 is not available for reading
x2 is not available for reading
=== Site 5 ===
x20: 200 at site 5
x16: 160 at site 5
x14: 140 at site 5
x10: 100 at site 5
x8: 80 at site 5
x6: 60 at site 5
x4: 40 at site 5
x18: 180 at site 5
x12: 120 at site 5
x2: 20 at site 5
=== Site 6 ===
x20: 200 at site 6
x14: 140 at site 6
x15: 150 at site 6
x10: 100 at site 6
x8: 80 at site 6
x6: 60 at site 6
x16: 160 at site 6
x5: 50 at site 6
x4: 40 at site 6
x18: 180 at site 6
x12: 120 at site 6
x2: 20 at site 6
=== Site 7 ===
x20: 200 at site 7
x16: 160 at site 7
x14: 140 at site 7
x10: 100 at site 7
x8: 80 at site 7
x6: 60 at site 7
x4: 40 at site 7
x18: 180 at site 7
x12: 120 at site 7
x2: 20 at site 7
=== Site 8 ===
x17: 170 at site 8
x16: 160 at site 8
x14: 140 at site 8
x10: 100 at site 8
x8: 80 at site 8
x20: 200 at site 8
x7: 70 at site 8
x6: 60 at site 8
x4: 40 at site 8
x18: 180 at site 8
x12: 120 at site 8
x2: 20 at site 8
=== Site 9 ===
x20: 200 at site 9
x16: 160 at site 9
x14: 140 at site 9
x10: 100 at site 9
x8: 80 at site 9
x6: 60 at site 9
x4: 40 at site 9
x18: 180 at site 9
x12: 120 at site 9
x2: 20 at site 9
=== Site 10 ===
x16: 160 at site 10
x19: 190 at site 10
x14: 140 at site 10
x10: 100 at site 10
x20: 200 at site 10
x9: 90 at site 10
x8: 80 at site 10
x6: 60 at site 10
x4: 40 at site 10
x18: 180 at site 10
x12: 120 at site 10
x2: 20 at site 10
Instruction can't be recognized
T1 read the value 20 of variable x2
T1 write the value 10 of variable x2
T1 write the value 10 of variable x2
T1 write the value 10 of variable x2
T1 write the value 10 of variable x2
T1 write the value 10 of variable x2
T1 write the value 10 of variable x2
T1 write the value 10 of variable x2
T1 committed
DEBUG| Clearing site locks for T1 variable: x2
Clearing locks for T1 variable: x2
DEBUG| Clearing site locks for T1 variable: x2
Clearing locks for T1 variable: x2
T3 committed
=== Site 1 ===
This site is down
=== Site 2 ===
x20: 200 at site 2
x16: 160 at site 2
x14: 140 at site 2
x8: 80 at site 2
x6: 60 at site 2
x11: 110 at site 2
x4: 40 at site 2
x18: 180 at site 2
x12: 120 at site 2
x2: 10 at site 2
x10: 100 at site 2
x1: 10 at site 2
=== Site 3 ===
This site is down
=== Site 4 ===
x20 is not available for reading
x16 is not available for reading
x14 is not available for reading
x10 is not available for reading
x6 is not available for reading
x4 is not available for reading
x13: 130 (available at site 4 for reading as it is the only copy or has been written after recovery)
x8 is not available for reading
x3: 30 (available at site 4 for reading as it is the only copy or has been written after recovery)
x18 is not available for reading
x12 is not available for reading
x2 is not available for reading
=== Site 5 ===
x20: 200 at site 5
x16: 160 at site 5
x14: 140 at site 5
x10: 100 at site 5
x8: 80 at site 5
x6: 60 at site 5
x4: 40 at site 5
x18: 180 at site 5
x12: 120 at site 5
x2: 10 at site 5
=== Site 6 ===
x20: 200 at site 6
x14: 140 at site 6
x15: 150 at site 6
x10: 100 at site 6
x8: 80 at site 6
x6: 60 at site 6
x16: 160 at site 6
x5: 50 at site 6
x4: 40 at site 6
x18: 180 at site 6
x12: 120 at site 6
x2: 10 at site 6
=== Site 7 ===
x20: 200 at site 7
x16: 160 at site 7
x14: 140 at site 7
x10: 100 at site 7
x8: 80 at site 7
x6: 60 at site 7
x4: 40 at site 7
x18: 180 at site 7
x12: 120 at site 7
x2: 10 at site 7
=== Site 8 ===
x17: 170 at site 8
x16: 160 at site 8
x14: 140 at site 8
x10: 100 at site 8
x8: 80 at site 8
x20: 200 at site 8
x7: 70 at site 8
x6: 60 at site 8
x4: 40 at site 8
x18: 180 at site 8
x12: 120 at site 8
x2: 10 at site 8
=== Site 9 ===
x20: 200 at site 9
x16: 160 at site 9
x14: 140 at site 9
x10: 100 at site 9
x8: 80 at site 9
x6: 60 at site 9
x4: 40 at site 9
x18: 180 at site 9
x12: 120 at site 9
x2: 10 at site 9
=== Site 10 ===
x16: 160 at site 10
x19: 190 at site 10
x14: 140 at site 10
x10: 100 at site 10
x20: 200 at site 10
x9: 90 at site 10
x8: 80 at site 10
x6: 60 at site 10
x4: 40 at site 10
x18: 180 at site 10
x12: 120 at site 10
x2: 10 at site 10
Starting T4
Starting T5
Starting read only transaction T6
Site: 2 failed
Site: 4 failed
Site: 5 failed
Site: 6 failed
Site: 7 failed
Site: 8 failed
Site: 9 failed
Site: 10 failed
Site: 2 recovered
T4 is waiting on x2
T5 got write lock on x2
DEBUG| T4 did not get read lock on x2 site: 2
T4 is blocked by T5 on x2
T5 write the value 100 of variable x2
T5 committed
DEBUG| Clearing site locks for T5 variable: x2
Clearing locks for T5 variable: x2
T4 got read lock on x2 having value 100
T4 already has a read lock on x2
Though the site holding x1 is recovering, T4 got read lock on x1 having value 10 since its the only copy
=== Site 1 ===
This site is down
=== Site 2 ===
x20 is not available for reading
x16 is not available for reading
x14 is not available for reading
x8 is not available for reading
x6 is not available for reading
x11: 110 (available at site 2 for reading as it is the only copy or has been written after recovery)
x4 is not available for reading
x18 is not available for reading
x12 is not available for reading
x2: 100 (available at site 2 for reading as it is the only copy or has been written after recovery)
x10 is not available for reading
x1: 10 (available at site 2 for reading as it is the only copy or has been written after recovery)
=== Site 3 ===
This site is down
=== Site 4 ===
This site is down
=== Site 5 ===
This site is down
=== Site 6 ===
This site is down
=== Site 7 ===
This site is down
=== Site 8 ===
This site is down
=== Site 9 ===
This site is down
=== Site 10 ===
This site is down
Site: 4 recovered
Site: 2 failed
T4 aborted as site 2 failed
T4 aborted as site 2 failed
=== Site 1 ===
This site is down
=== Site 2 ===
This site is down
=== Site 3 ===
This site is down
=== Site 4 ===
x20 is not available for reading
x16 is not available for reading
x14 is not available for reading
x10 is not available for reading
x6 is not available for reading
x4 is not available for reading
x13: 130 (available at site 4 for reading as it is the only copy or has been written after recovery)
x8 is not available for reading
x3: 30 (available at site 4 for reading as it is the only copy or has been written after recovery)
x18 is not available for reading
x12 is not available for reading
x2 is not available for reading
=== Site 5 ===
This site is down
=== Site 6 ===
This site is down
=== Site 7 ===
This site is down
=== Site 8 ===
This site is down
=== Site 9 ===
This site is down
=== Site 10 ===
This site is down
DEBUG| Clearing site locks for T4 variable: x1
Clearing locks for T4 variable: x1
DEBUG| Clearing site locks for T4 variable: x2
Clearing locks for T4 variable: x2
Starting T7
T7 got write lock on x4
T7 write the value 75 of variable x4
T7 committed
DEBUG| Clearing site locks for T7 variable: x4
Clearing locks for T7 variable: x4
T6 committed
=== Site 1 ===
This site is down
=== Site 2 ===
This site is down
=== Site 3 ===
This site is down
=== Site 4 ===
x20 is not available for reading
x16 is not available for reading
x14 is not available for reading
x10 is not available for reading
x6 is not available for reading
x4: 75 (available at site 4 for reading as it is the only copy or has been written after recovery)
x13: 130 (available at site 4 for reading as it is the only copy or has been written after recovery)
x8 is not available for reading
x3: 30 (available at site 4 for reading as it is the only copy or has been written after recovery)
x18 is not available for reading
x12 is not available for reading
x2 is not available for reading
=== Site 5 ===
This site is down
=== Site 6 ===
This site is down
=== Site 7 ===
This site is down
=== Site 8 ===
This site is down
=== Site 9 ===
This site is down
=== Site 10 ===
This site is down
